CHANGELOG.md

```diff
@@ -24,7 +24,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - The `Status` type was added to the `go.opentelemetry.io/otel/sdk/trace` package to represent the status of a span. (#1874)
 - The `SpanStub` type and its associated functions were added to the `go.opentelemetry.io/otel/sdk/trace/tracetest` package.
   This type can be used as a testing replacement for the `SpanSnapshot` that was removed from the `go.opentelemetry.io/otel/sdk/trace` package. (#1873)
-
+- Adds `otlphttp.WithHTTPTransport`, `otlphttp.WithTracesHTTPTransport` and `otlphttp.WithMetricsHTTPTransport` options for 
+  configuring a custom HTTP transport used to deliver the traces and metric data to the otlp/http endpoint. (#1858)
```

Comment:
```suggestion
  configuring a custom HTTP transport used to deliver the traces and metric data to the otlp/http endpoint. (#1881)
```

this should reference the PR, not the issue.  Referencing the PR helps people jump directly to the code that implemented the change.